<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParSetOne</class>
 <widget class="QWidget" name="ParSetOne">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#groupBox_3 QLabel,
#groupBox QLabel,
#groupBox_2 QLabel,
#label_weight{
	border: 1px solid rgba(220, 220, 220, 0.8);
}
#groupBox_3 QDoubleSpinBox,
#groupBox QDoubleSpinBox,
#groupBox_2 QDoubleSpinBox,
#groupBox_3 QSpinBox,
#groupBox QSpinBox,
#groupBox_2 QSpinBox{
	margin-right: 2px;
	margin-left: 2px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_10">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>养殖设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="2">
         <widget class="QLabel" name="label_33">
          <property name="text">
           <string>养殖数量</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QLabel" name="label_36">
          <property name="text">
           <string>单只体重(kg)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QSpinBox" name="spinBox_matrix_count">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>100000000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>畜禽类别</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QSpinBox" name="spinBox_current_days">
          <property name="minimum">
           <number>-3</number>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox_matrix_category">
          <item>
           <property name="text">
            <string>肉鸡种鸡</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>肉鸡商品代</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>蛋鸡种鸡</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>蛋鸡商品代</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>黄鸡种鸡</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>黄鸡商品代</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>猪群</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>兔群</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>其他品系</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label_35">
          <property name="text">
           <string>当前日龄(d)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QLabel" name="label_weight">
          <property name="text">
           <string>***</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>215</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>目标温度</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>日龄(d)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox_aim_day1">
          <property name="minimum">
           <number>-3</number>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="spinBox_aim_day2">
          <property name="minimum">
           <number>-3</number>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QSpinBox" name="spinBox_aim_day3">
          <property name="minimum">
           <number>-1000</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QSpinBox" name="spinBox_aim_day4">
          <property name="minimum">
           <number>-1000</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QSpinBox" name="spinBox_aim_day5">
          <property name="minimum">
           <number>-1000</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QSpinBox" name="spinBox_aim_day6">
          <property name="minimum">
           <number>-1000</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QSpinBox" name="spinBox_aim_day7">
          <property name="minimum">
           <number>-1000</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>目标温度(°C)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_aim_t1"/>
        </item>
        <item row="1" column="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_aim_t2"/>
        </item>
        <item row="1" column="3">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_aim_t3"/>
        </item>
        <item row="1" column="4">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_aim_t4"/>
        </item>
        <item row="1" column="5">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_aim_t5"/>
        </item>
        <item row="1" column="6">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_aim_t6"/>
        </item>
        <item row="1" column="7">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_aim_t7"/>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>108</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>最小通风设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>夜间单位体重通风量折扣系数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_discount_coefficient">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>0.500000000000000</double>
          </property>
          <property name="maximum">
           <double>1.500000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_m_air_unit_v1"/>
        </item>
        <item row="1" column="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_m_air_matrix_w2"/>
        </item>
        <item row="4" column="4">
         <widget class="QLabel" name="label_m_air_a4">
          <property name="text">
           <string>***</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QSpinBox" name="spinBox_m_air_day3"/>
        </item>
        <item row="2" column="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_m_air_unit_v2"/>
        </item>
        <item row="4" column="5">
         <widget class="QLabel" name="label_m_air_a5">
          <property name="text">
           <string>***</string>
          </property>
         </widget>
        </item>
        <item row="3" column="7">
         <widget class="QLabel" name="label_m_air_c7">
          <property name="text">
           <string>***</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_m_air_c1">
          <property name="text">
           <string>***</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_m_air_matrix_w1"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;单位体重通风量(m&lt;span style=&quot; vertical-align:super;&quot;&gt;3&lt;/span&gt;/h/kg)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_m_air_matrix_w4"/>
        </item>
        <item row="3" column="5">
         <widget class="QLabel" name="label_m_air_c5">
          <property name="text">
           <string>***</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_m_air_c2">
          <property name="text">
           <string>***</string>
          </property>
         </widget>
        </item>
        <item row="2" column="7">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_m_air_unit_v7"/>
        </item>
        <item row="1" column="3">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_m_air_matrix_w3"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;计算最小风量(m&lt;span style=&quot; vertical-align:super;&quot;&gt;3&lt;/span&gt;/h)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_m_air_unit_v5"/>
        </item>
        <item row="1" column="5">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_m_air_matrix_w5"/>
        </item>
        <item row="3" column="6">
         <widget class="QLabel" name="label_m_air_c6">
          <property name="text">
           <string>***</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>日龄(d)</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_m_air_a1">
          <property name="text">
           <string>***</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>母畜禽单只体重(kg)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>自动</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="4">
         <widget class="QSpinBox" name="spinBox_m_air_day4"/>
        </item>
        <item row="0" column="5">
         <widget class="QSpinBox" name="spinBox_m_air_day5"/>
        </item>
        <item row="4" column="6">
         <widget class="QLabel" name="label_m_air_a6">
          <property name="text">
           <string>***</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="label_m_air_a3">
          <property name="text">
           <string>***</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox_m_air_day1"/>
        </item>
        <item row="2" column="3">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_m_air_unit_v3"/>
        </item>
        <item row="3" column="3">
         <widget class="QLabel" name="label_m_air_c3">
          <property name="text">
           <string>***</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QSpinBox" name="spinBox_m_air_day6"/>
        </item>
        <item row="1" column="6">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_m_air_matrix_w6"/>
        </item>
        <item row="1" column="7">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_m_air_matrix_w7"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;实际最小风量(m&lt;span style=&quot; vertical-align:super;&quot;&gt;3&lt;/span&gt;/h)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_m_air_unit_v4"/>
        </item>
        <item row="2" column="6">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_m_air_unit_v6"/>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="spinBox_m_air_day2"/>
        </item>
        <item row="4" column="7">
         <widget class="QLabel" name="label_m_air_a7">
          <property name="text">
           <string>***</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_m_air_a2">
          <property name="text">
           <string>***</string>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QSpinBox" name="spinBox_m_air_day7"/>
        </item>
        <item row="3" column="4">
         <widget class="QLabel" name="label_m_air_c4">
          <property name="text">
           <string>***</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>折扣时间</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>第一组</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="timeEdit_dc_1">
              <property name="displayFormat">
               <string>HH:mm</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="timeEdit_dc_2">
              <property name="displayFormat">
               <string>HH:mm</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>第三组</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="timeEdit_dc_5">
              <property name="displayFormat">
               <string>HH:mm</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="timeEdit_dc_6">
              <property name="displayFormat">
               <string>HH:mm</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>第二组</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="timeEdit_dc_3">
              <property name="displayFormat">
               <string>HH:mm</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="timeEdit_dc_4">
              <property name="displayFormat">
               <string>HH:mm</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>第四组</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="timeEdit_dc_7">
              <property name="displayFormat">
               <string>HH:mm</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="timeEdit_dc_8">
              <property name="displayFormat">
               <string>HH:mm</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>通风级别限制</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>日龄</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox_level_d1"/>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="spinBox_level_d2"/>
        </item>
        <item row="0" column="3">
         <widget class="QSpinBox" name="spinBox_level_d3"/>
        </item>
        <item row="0" column="4">
         <widget class="QSpinBox" name="spinBox_level_d4"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>上限</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox_level_up1"/>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="spinBox_level_up2"/>
        </item>
        <item row="1" column="3">
         <widget class="QSpinBox" name="spinBox_level_up3"/>
        </item>
        <item row="1" column="4">
         <widget class="QSpinBox" name="spinBox_level_up4"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>下限</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spinBox_level_down1"/>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="spinBox_level_down2"/>
        </item>
        <item row="2" column="3">
         <widget class="QSpinBox" name="spinBox_level_down3"/>
        </item>
        <item row="2" column="4">
         <widget class="QSpinBox" name="spinBox_level_down4"/>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>14</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
